### PR TITLE
Release v2.0.1

### PR DESCRIPTION
**Pull request by rosik**
And also:

- Move CI to Github Actions
- Remove Travis CI
- Remove Gitlab CI
- Remove unnecessary files

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      - uses: actions/checkout@v2
+      - uses: rosik/setup-tarantool@v1
+        with: {tarantool-version: '1.10'}
+
+      - name: Upload rockspec
+        run: |
+          TAG=${GITHUB_REF##*/}
+          ./release.sh $TAG
+          ./upload.sh release/luarapidxml-${TAG}-1.rockspec
+          ./upload.sh release/luarapidxml-${TAG}-1.src.rock
```
Comment (olegrok) [inline .github/workflows/publish.yml]: For future, seems we need separate action that accepts rock name and credentials and push rockspec.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  push:
+  workflow_dispatch:
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        runs-on: [ubuntu-20.04]
+        tarantool: ['1.10', '2.4', '2.5', '2.6']
```
Comment (olegrok) [inline .github/workflows/test.yml]: Do you really need all of them for luarapidxml? I think 1.10 and 2.6 is more than enough.

**Review by olegrok — APPROVED** [review]
Thanks for your patch, please consider small comments below.

File: .github/workflows/test.yml (hunk 2 above)
Comment (Totktonada) [reply .github/workflows/test.yml]: BTW, can we run testing on 2.7-live to spot possible problems earlier?

Comment (rosik) [reply .github/workflows/test.yml]: Discussed the matrix in chat. The jobs are parallel, all take 50 sec. There's nothing to save.
2.7 added.

**Pull request merged**